docker/dev_env/Dockerfile

```diff
@@ -42,16 +45,18 @@ RUN dnf -y install dnf-plugins-core \
     gcc \
     just \
     which \
-    pipx \
-    nodejs \
+    nodejs npm \
+    python3.12 pipx \
     docker-ce docker-ce-cli containerd.io docker-buildx-plugin docker-compose-plugin \
   && pipx install \
     pdm pipenv  \
     pre-commit \
-  && npm install -g \
+    && npm install -g \
```

Comment:
Was this additional indentation accidental or intentional?

Reply:
Unintentional, I'll fix this.